docs/root/configuration/http/http_filters/_include/grpc-json-reverse-transcoder-filter.yaml

```diff
@@ -0,0 +1,54 @@
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 51051
```

Comment:
i would suggest using port 10000 for this as that is the common default - it would also make it less confusing in relation to the upstream port

Reply:
Done